now.json

```diff
@@ -8,5 +8,8 @@
       "use": "@now/static-build",
       "config": {"distDir": "packages/paste-website/public"}
     }
+  ],
+  "alias": [
```

Comment:
Is this needed?  According to the docs this gets ignored if configured through the dashboard https://zeit.co/docs/v2/advanced/configuration/#alias

Reply:
I added this as a “just in case” the domain didn’t alias automatically (I had some issues with both my sites previously). I can close this now